ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/432__data-science-pipelines-tekton.robot

```diff
@@ -35,7 +35,7 @@ End To End Pipeline Workflow Using Kfp Tekton
     Install DataSciencePipelinesApplication CR    ${project}
     ${status}    Login Using User And Password    ${username}    ${password}    ${project}
     Should Be True    ${status} == 200    DSP routing is working
```

Comment:
"DSP routing is working should be replaced by "Could not login to the Data Science Pipelines Rest API" or similar  (this is the message shown when not true)